src/ontology/cl-edit.owl

```diff
@@ -27509,6 +27493,18 @@ SubClassOf(obo:CL_4023013 obo:CL_0000679)
 SubClassOf(obo:CL_4023013 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956))
 SubClassOf(obo:CL_4023013 ObjectSomeValuesFrom(obo:RO_0013001 obo:UBERON_0010225))
 
+# Class: obo:CL_4023014 (L5 Vip expressing cortical GABAergic interneuron (Mus musculus))
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:26420784") Annotation(oboInOwl:hasDbXref "PMID:26612957") obo:IAO_0000115 obo:CL_4023014 "A Vip GABAergic cortical interneuron found in L5. L5 Vip cells have mostly local morphology with some deep-projecting axons. They show only moderate resistance, comparable to that of Sst subclass and unlike typical Vip subclass cells that tend to show high input resistance. L5 Vip cells show particularly low resting membrane potential.")
```

Comment:
If the other defs. include 'in mouse', then this should as well. I am unfamiliar with the standard so cannot settle the issue.

Reply:
You do make a good point - that entry is just reshuffled, but we should probably add in Mouse to all that are mouse specific. Will write a separate ticket on this.